packages/main/test/specs/Link.spec.js

```diff
@@ -61,4 +60,11 @@ describe("General API", () => {
 		const url = await browser.getUrl();
 		assert.notInclude(url, "https://www.google.com", "URL is not google");
 	});
+
+	it("Collabsible element has aria expanded attribute", async () => {
+		const link = await browser.$("#collapse");
+
+		await link.click();
+		assert.strictEqual(await link.getAttribute("aria-expanded"), "false", "Text is collapsed");
```

Comment:
this should be true?

Reply:
By default the text is expanded after press it gets collapsed. I've added one additional assert.